app/assets/stylesheets/application.css

```diff
@@ -199,6 +199,7 @@ body {
 	.job {
 		border-bottom: 1px solid #ccc;
 		padding: 6px;
+    line-height: 20px;
```

Comment:
I definitely wouldn't go with a smaller icon, it's difficult enough to see what it is as is.